src/amo/components/AddonBadges/index.js

```diff
@@ -26,49 +20,27 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
-  _config: typeof config,
   clientApp: string,
   i18n: I18nType,
 |};
 
 export class AddonBadgesBase extends React.Component<InternalProps> {
-  static defaultProps = {
-    _config: config,
-  };
-
   render() {
-    const { _config, addon, clientApp, i18n } = this.props;
+    const { addon, clientApp, i18n } = this.props;
 
     if (!addon) {
       return null;
     }
 
-    const getFeaturedText = (addonType: string) => {
-      switch (addonType) {
-        case ADDON_TYPE_EXTENSION:
-          return i18n.gettext('Featured Extension');
-        case ADDON_TYPE_STATIC_THEME:
-        case ADDON_TYPE_THEME:
-          return i18n.gettext('Featured Theme');
-        default:
-          return i18n.gettext('Featured Add-on');
-      }
-    };
-
     const isIncompatible =
       addon.type === ADDON_TYPE_EXTENSION &&
       isQuantumCompatible({ addon }) === false;
 
     return (
       <div className="AddonBadges">
-        {_config.get('enableFeatureRecommendedBadges') &&
-        addon.is_recommended &&
-        clientApp !== CLIENT_APP_ANDROID ? (
+        {addon.is_recommended && clientApp !== CLIENT_APP_ANDROID ? (
           <RecommendedBadge size="large" />
         ) : null}
-        {addon.is_featured && !_config.get('enableFeatureRecommendedBadges') ? (
```

Comment:
Should we also remove `is_featured` in the type definition (and in the reducer too)?

- https://github.com/mozilla/addons-frontend/blob/44511f71fb472c7de1ad37632c3c13ff545c3f2f/src/core/reducers/addons.js#L212
- https://github.com/mozilla/addons-frontend/blob/44511f71fb472c7de1ad37632c3c13ff545c3f2f/src/core/types/addons.js#L94

Reply:
I suppose we should. I actually intentionally left some support for `is_featured` in place because it still exists in the API, and I wasn't sure we wanted to remove something that is available via the API. But I suppose if we have no intention of ever using it we should remove it. This would also allow the back end to remove this from the API in the future should we wish to do so.